src/themes/legacy/_colors_dark.scss

```diff
@@ -48,4 +48,4 @@ $euiColorChartBand: tint($euiColorLightestShade, 2.5%) !default;
 
 // Shadows
 $euiShadowColor: #000 !default;
-$euiShadowColorLarge: #000 !default;
+$euiShadowColor: #000 !default;
```

Comment:
Can this be removed?

Reply:
No we still need the Sass version to work.

Reply:
But L50 is identical?

Reply:
Oh sorry, during my find and replace, I didn't think of these lines. Yeah I'll change

Reply:
In fact, I actually don't think we need to redeclare this variable at all in dark mode since ink = #000 in all themes